fashion_ai_bounded_autonomy/tasks.py

```
@@ -0,0 +1,479 @@
+            - "error" (str): Error message when status is "failed".
+    """
+    try:
+        from fashion_ai_bounded_autonomy.performance_tracker import PerformanceTracker
+
+        tracker = PerformanceTracker()
+
+        # Get current KPIs
+        summary = tracker.get_kpi_summary()
+
+        logger.info("📸 Captured performance snapshot")
+        return {"status": "completed", "metrics": summary}
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)